app/schemas/project_preference_update_project_settings_schema.rb

```diff
@@ -0,0 +1,21 @@
+class ProjectPreferenceUpdateSchema < JsonSchema
```

Comment:
this stuff, https://github.com/camallen/Panoptes/commit/94b915db3bf0340a8c2e097a038ddc5393a462b7